transit_odp/publish/templates/publish/dataset_detail/timetable_journey_stop.html

```diff
@@ -0,0 +1,99 @@
+{% load i18n %}
+{% load query_transform %}
+{% load parse_dictionary %}
+
+
+{% if  timetable_details.df.columns|length  <= 5 %}
+<table class="govuk-table govuk-table--small-text-until-tablet timetables_govuk_table">
+{% else %}
+<table class="govuk-table govuk-table--small-text-until-tablet">
+{% endif %}
+  <caption class="govuk-table__caption govuk-table__caption--m">{% trans timetable_details.journey_name %}</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      {% for col in timetable_details.df.columns %}
+        <th scope="col" class="govuk-table__header ">{{col}}</th>
+    {% endfor %}
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    {% for idx, row in timetable_details.df.iterrows %}
+      <tr class="govuk-table__row">
+        {% for col_name, cell in row.items %}
+        <td class="govuk-table__cell gov-table-vr">          
+            {% if col_name == 'Journey Code'  %}
+              {% with key=cell|concatenate:"_"|concatenate:idx %}              
+              <a class="tooltip govuk-link" title="More information" id="stop-{{direction}}-{{idx}}" onclick="BODSFrontend.showTooltip(event,'{{idx}}', '{{direction}}');"  >{{cell}}                
+                <p class="tooltiptext" id="tooltip-{{direction}}-{{idx}}">
```

Comment:
Can the width of the tooltip box match what is displayed in the figma design - the text need is not wrapped to the next line